test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -18,6 +18,273 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : TagHelperRewritingTestBase
     {
+        public static TheoryData InvalidHtmlScriptBlockData
+        {
+            get
+            {
+                var factory = CreateDefaultSpanFactory();
+                var blockFactory = new BlockFactory(factory);
+
+                return new TheoryData<string, MarkupBlock>
+                {
+                    {
+                        "<script type><input /></script>",
```

Comment:
Supposed to be invalid hence the `InvalidHtmlScriptBlockData`. This is just making sure Razor doesn't think it's valid :smile: